student.rb

```diff
@@ -0,0 +1,9 @@
+class Student < Person
+  def initialize(classroom = '10th', play_hooky = "¯\(ツ)/¯")
+    super()
+    @classroom = classroom
+    @play_hooky = play_hooky
+  end
```

Comment:
- [ ] Constructor extends parent's constructor by adding @classroom and a parameter for it, something like so: 
```ruby
 def initialize(age, classroom, name = "Unknown", parent_permission = true)  
    super(age, name, parent_permission)
    @classroom = classroom
  end
```
Checkout this [article](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/class_and_object_for_irb.md#constructor-and-instance-vars) and kindly fix that.